src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+class Duke {
+
+    public static void main(String[] args) throws DukeException{
```

Comment:
You might need to leave a space after _DukeException_ 
```suggestion
    public static void main(String[] args) throws DukeException {
```